sdv/sdv.py

```diff
@@ -159,4 +162,10 @@ def load(cls, path):
                 Path from which to load the SDV instance.
         """
         with open(path, 'rb') as f:
-            return pickle.load(f)
+            model = pickle.load(f)
+            warning_str = generate_version_mismatch_warning(
+                getattr(model, '_package_versions', None))
+            if warning_str:
+                warnings.warn(warning_str)
```

Comment:
Is there any reason not to move this whole chunk to utils and just call a method that raises the warning if necessary?

Reply:
I like to have the warning originating from the actual method, not a utils method. That way it's more obvious if a method raises a warning / error.